packages/components/src/KaizenProvider/_docs/internationalization-in-kaizen.mdx

```diff
@@ -12,7 +12,7 @@ If you need guidance on translating your app check, the `@cultureamp/i18n-react-
 ## What Kaizen internationalization covers
 
 While most Kaizen components receive display text and accessibility information through props, some have text built in. For some of these cases, Kaizen provides translation support. 
```

Comment:
So we do or don't still provide translation support?

Reply:
Not any more. Reworded it to reflect that